### DISCUSSION TITLE
fasttrack debian.net 2 repositories - could be there is a problem if the source was added as https?

### DISCUSSION THREAD

**Discussion by ghost** [SpiralLinux/SpiralLinux-project#197] [General]
Discussion reletes to: 2 fastrack repositories, adding sources as https, and possibly also networking with new kernel, firmware.

I know this is very vogue and very non accurate (but maybe still useful hence posted), and I am not the ideal person to relate to that, don't have the right/needed information here, but there is something I noticed which may be important.

Both fasttrack repositories of debian.net seemingly are not working for me on my temporary (stable version) USB live system - noticed that after some updates I've made to the system. 
There seem to errors on check and warnings on upgrades, related to https or connection refused (I don't have this error available now).
Could this be an https issue - an error while a source can't really be added as an https source? (they have sources for both https and http in case https can't be added to sources, and the site refered to the possibility that in some cases one can't use https sources)
 
If yes, I further speculate: Could that be also related to some issues others encountered? (even possibly networking issue due to missing packages, though I don't know if those are related to those repositories)
Noting I don't have a networking issue on that USB live system myself though I probably do have one of the affected hardare using that wifi firmware (I can't bring more details at the moment but will ask how to check things again if that detail is needed.)
This also leads to a speculation things might get messed up with a kernel update (one which I didn't do) possibly without those repositories available then to mend the networking firmware package issue - or a similar thing.

**Comment by ghost** [comment]
Update: I changed those to http and now updates works better and without errors or warnings and synaptic and seemingly the system is (more) responding. Your milage may vary - this is not a recommendation.

However allegedly there is no relation to any connectivity issue as those packages are allegdly non affected by this as far as I could see.
As others said, though, kernel update may be related to the connectivity issue, just not this source topic.

**Reply by geckolinux** [reply]
Hi there,

> Both fasttrack repositories of debian.net seemingly are not working for me

This appears to be a random server error, I've seen it too. It may or may not be related to the `https://` links, but that's what they recommend:
https://fasttrack.debian.net/#instructions

> This also leads to a speculation things might get messed up with a kernel update (one which I didn't do) possibly without those repositories available then to mend the networking firmware package issue - or a similar thing.

No, the only packages pre-installed from FastTrack are related to VirtualBox.

**Comment by bittin** [comment]
@Visiting-Newbie @geckolinux 

Fasttrack was down a bit yesterday, but should be up now again so you are correct

**Reply by ghost** [reply]
Thank you both for the information, it seems to work now indeed.
